guides/source/configuring.md

```diff
@@ -116,8 +116,7 @@ numbers. It also filters out sensitive values of database columns when call `#in
 
 * `config.log_formatter` defines the formatter of the Rails logger. This option defaults to an instance of `ActiveSupport::Logger::SimpleFormatter` for all modes. If you are setting a value for `config.logger` you must manually pass the value of your formatter to your logger before it is wrapped in an `ActiveSupport::TaggedLogging` instance, Rails will not do it for you.
 
-* `config.log_level` defines the verbosity of the Rails logger. This option
-defaults to `:debug` for all environments. The available log levels are: `:debug`,
+* `config.log_level` defines the verbosity of the Rails logger. This option defaults to `:debug` for all modes except production, where it defaults to `:info`. The available log levels are: `:debug`,
```

Comment:
```suggestion
* `config.log_level` defines the verbosity of the Rails logger. This option defaults to `:debug` for all environments except production, where it defaults to `:info`. The available log levels are: `:debug`,
```

I know we use "mode" elsewhere in this guide, but environment is the correct term.

Can you also rewrap at 80 characters?

Reply:
@eugeneius I must admit I am a little bit confused about the wrapping part. Looking at the other paragraphs above and below I don't see them wrapped at any char length. Or are you asking me to pull the line up that is wrapped after
> are: `:debug`,

Reply:
Before this change the paragraph was wrapped at 80 characters, I was suggesting to apply that same formatting to the new version. You're right that it's not applied consistently throughout the file though, so pulling the next line up is also acceptable.